client/android/client.go

```diff
@@ -107,6 +104,11 @@ func (c *Client) Stop() {
 	c.ctxCancel()
 }
 
+// SetTraceLogLevel configure the logger to trace level
+func (c *Client) SetTraceLogLevel() {
```

Comment:
This function is never used

Reply:
It is an exported function. We are using it from external repo. The sense of this package is to export functions for Java code. Please check this pull request: https://github.com/netbirdio/react-native-app/pull/14